### DISCUSSION TITLE
How to Configure Scheduled Jobs for Tenants not Landlord?

### DISCUSSION THREAD

**Discussion by gondel07** [spatie/laravel-multitenancy#521] [Q&A]
I have scheduled a job in console/kernel.php

```php
Log::info(Tenant::current());
if (Tenant::current()) {
     $schedule->job(new Testjob)->everyTwoMinutes();       
}
```

but when i run 
`* * * * *  php artisan tenants:artisan schedule:work`

in laravel.log i Tenant::current() gives me NULL 

my TestJob is

```php
class Testjob implements ShouldQueue,TenantAware
{
    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;

    public function handle()
    {
        Log::info('TenantNoopJob - '. Tenant::current()->domain .' - '. now());
    }
}
```

i followed the 

what i made wrong?
i have question how Scheduler be tenant aware?

i followed the the [#82 ](https://github.com/spatie/laravel-multitenancy/issues/82) 
but not understand how current tenant id is found 

@freekmurze @masterix21 please help

**Comment by masterix21** [comment]
Please create a public repository with your code so we can help you :)
